### DISCUSSION TITLE
Vernet v1.0.4 adds ”Home” app permission.

### DISCUSSION THREAD

**Discussion by conceptrat** [osociety/vernet#144] [General]
On the latest version ”v1.0.4”, as of this post, the app adds the ”Home” permission.  This causes the current ”Home” app to lose it's status as the default ”Home” app and results in a dialog box popping up requesting the choosing of a default ” Home' app again.

Why does Vernet need the ”Home” app permission?

![Screenshot_20240327171831](https://github.com/osociety/vernet/assets/831863/5cb9a62c-d728-4df9-ae92-6bcd1d8b5064)

**Comment by git-elliot** [comment]
I recently add external_app_links package and I didn't notice that I accidently added home app intent because of it's instructions. Now it is removed and next build will not have this problem.

**Reply by conceptrat** [reply]
Although it wasn't a "show stopper", thank you for the explanation and fixing this.  Much appreciated 👍

**Reply by git-elliot** [reply]
But I feel, it can be annoying sometimes for new users.
